lib/FetchEnvs.js

```diff
@@ -133,7 +133,7 @@ module.exports = class FetchEnvs {
     if (krm) {
       try {
         response = await this.api({
-          uri: krm.uri({ namespace, kind, name }),
+          uri: krm.uri({ namespace, name }),
```

Comment:
it this just not needed, or was it causing a problem?

Reply:
just not needed. `krm.uri()` doesnt use it and i didnt want to be confused later on